MirrorProvider/MirrorProvider.Windows/WindowsFileSystemVirtualizer.cs

```diff
@@ -316,6 +320,22 @@ private void OnFileModifiedOrDeleted(string relativePath, bool isDirectory, bool
             Console.WriteLine($"OnFileModifiedOrDeleted: `{relativePath}`, isDirectory: {isDirectory}, isModfied: {isFileDeleted}, isDeleted: {isFileDeleted}");
         }
 
+        private void OnFileRenamed(
+            string relativePath,

```

Comment:
relativeSourcePath?

Reply:
See other comment as well.  Currently these names match the names used in the ProjFS API, let me know if you think it's better to keep them consistent or change them.